public_html/wp-content/mu-plugins/blocks/webpack.config.js

```diff
@@ -1,67 +1,38 @@
 const path = require( 'path' );
-const webpack = require( 'webpack' );
 const MiniCssExtractPlugin = require( 'mini-css-extract-plugin' );
+const defaultConfig = require( '@wordpress/scripts/config/webpack.config' );
 
-const NODE_ENV = process.env.NODE_ENV || 'development';
-const SOURCE_MAPS = process.env.SOURCE_MAPS ? true : false;
-
-const externals = {
-	react       : 'React',
-	'react-dom' : 'ReactDOM',
-	lodash      : 'lodash',
-};
-
-const webpackConfig = {
-	// Must be 'none', 'production', or 'development'.
-	mode: NODE_ENV === 'production' ? 'production' : 'development',
-
-	optimization: {
-		minimize: true,
-	},
-
-	// Disabled by default because they make the re-build process take longer.
-	devtool: SOURCE_MAPS ? 'cheap-module-eval-source-map' : 'none',
+module.exports = {
+	...defaultConfig,
 
+	// We use a custom entry point since our source directory is not `src`.
 	entry: {
 		blocks: path.resolve( __dirname, 'source/blocks.js' ),
 	},
 
+	// Override the default filename to keep `min` in the name.
 	output: {
-		filename : '[name].min.js',
-		path     : path.resolve( __dirname, 'build' ),
+		...defaultConfig.output,
+		filename: '[name].min.js',
 	},
 
+	// We need to extend the module.rules & plugins to add the scss build process.
```

Comment:
It looks like this fully take cares of https://github.com/WordPress/gutenberg/issues/14801 for our repo, so that we don't have to wait for upstream to implement it? Is that right?

Reply:
Right, since the core webpack config doesn't support CSS, we set it up ourselves. If that ends up being implemented, we'll probably want to remove our config- I'm not sure how webpack would handle duplicate loaders.

Reply:
Awesome! Thanks :)